docker/nugus_sim/run.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import subprocess
+import sys
+
+import ruamel.yaml
+
+BINARIES_DIR = "/home/nubots/NUbots/binaries"
+CONFIG_DIR = BINARIES_DIR + "/config"
+
+ENV_VARS = ("ROBOCUP_ROBOT_ID", "ROBOCUP_TEAM_COLOR", "ROBOCUP_SIMULATOR_ADDR")
+
+
+def read_args() -> dict:
+    parser = argparse.ArgumentParser(description="Run a role for webots RoboCup")
+
+    parser.add_argument("role", help="The role to run")
+    parser.add_argument(
+        "--goalie", action=argparse.BooleanOptionalAction, default=False, help="Run the role with goalie behaviour"
```

Comment:
I have never seen this before, I usually just go with `store_true`
```suggestion
        "--goalie", action="store_true", default=False, help="Run the role with goalie behaviour"
```